rental_ui.rb

```diff
@@ -0,0 +1,54 @@
+class RentalUserInterface
+  def initialize
+    @rentals = []
+  end
+
+  def create_rental(books, people)
+    get_book_from_user(books)
+    book_num = gets.chomp
+
+    get_person_from_user(people)
+    person_num = gets.chomp
+
+    puts 'Date: '
+    date = gets.chomp
+
```

Comment:
- [ ] Kindly ensure user inputs and validation are handled by another method.

Reply:
thanks for the feedback